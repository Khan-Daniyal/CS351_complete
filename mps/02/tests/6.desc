mytr: file input with -d
