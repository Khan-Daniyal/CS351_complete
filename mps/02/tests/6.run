./mytr -d ' abcdefghijklmnopqrstuvwxyz' < tests/5.in
