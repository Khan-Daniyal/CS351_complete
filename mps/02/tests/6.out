ALICE’SADVENTURESINWONDERLAND

LC

THEMILLENNIUMFULCRUMEDITION3.0

CHAPTERI.DR-H

A
,:
,
,‘,’A‘
?’

S(,
),
-
,WR
.

TVERY;A
VERYR,‘O!
O!I!’(,
,
);RTOOKAWATCH
OUTOFITSWAISTCOAT-POCKET,,,
A,
-,
,,
,
-.

IA,
.

T-,
,A

.

E,,

.F,
,;
,
-;
.S
;‘ORANGEMARMALADE’,
:
,
.

‘W!’A,‘,I
!H’
!W,I’,I
!’(W.)

D,,.WNEVER!‘I
I’?’.‘I
.L:
,I--’(,,A
,
VERY,
,)
‘--,’--IL
LI’?’(AL,
L,.)

P.‘IITHROUGH
!H’
!TA,I--’(
WAS,,’
)‘--I
,.P,M’,NZA?’(
--CURTSEYING’
!D?)‘A
’!N,’
:I.’

D,,.T,A
.‘D’-,I!’
(D.)‘I’
-.D!I!T
,I’,,’
,.B,I?’AA
,,
,‘D?D?’,‘D
?’,,’,
’.S
,
D,,‘N,D,:
?’,!!
,.

A,:
,;
,WR,.
T:A,
,,‘O
,’!’S
,R:
,,
.

T,;
A,
,,
.

S-,
;,A’
;
,!,,
.H,
,,
:
,!

A,
-:
.H
,
,
;‘,’A,‘
.O,II
!II,I.’
F,,---,
A
.

T,
,,
:
,[‘
,’A,)
,‘DRINKME’
.

I‘D,’A
THAT.‘N,I’,’,‘
’“”’;
,
,WOULD
:,-
;
VERY,;
,‘,’
,.

H,NOT‘,’A
,,(,,
-,,-,,,
,).

‘W!’A;‘I
.’

A:,

.F,,
:
;‘,,’
A,‘,.I
I?’A
,
.

A,,
;,A!
,,
,
:,
,;
,
.

‘C,’!’A,
;‘I!’S
,(),

;
,
.
‘B’,’A,‘!
W,’ONE!’

S:
,,
‘EATME’.‘W,I’,’
A,‘,I;
,I;I’
,I’!’

S,,‘W?W
?’,
,
:,,A
---
,
.

S,.